bin/dry-run.rb

```diff
@@ -80,68 +81,120 @@
 # GitHub credentials with write permission to the repo you want to update
 # (so that you can create a new branch, commit and pull request).
 # If using a private registry it's also possible to add details of that here.
-credentials = []
+
+OPTIONS = {
```

Comment:
Maybe clearer as a global rather than a constant?

```suggestion
$options = {
```